pkg/reconciler/source/config_watcher.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
would that file be perhaps something for `pkg` repo?

Reply:
I thought about that, but this is really specific to knative sources, so perhaps is better to put it here and then move later to pkg if needed